repository_cli/cli/records.py

```diff
@@ -132,10 +133,94 @@ def delete_record(pid: str):
     click.secho(f"'{pid}', soft-deleted", fg="green")
 
 
+@rdmrecords.group()
+def pids():
+    """Management commands for record pids."""
+
+
+@pids.command("list")
+@option_pid(required=True)
+@with_appcontext
+def list_pids(pid: str):
+    """List record's pids.
+
+    example call:
+        invenio repository rdmrecords pids list -p <pid>
+    """
+    if not record_exists(pid):
+        click.secho(f"'{pid}', does not exist or is deleted", fg="red")
+        return
+
+    identity = get_identity()
+    service = get_records_service()
+    record_data = service.read(id_=pid, identity=identity).data.copy()
+    current_pids = record_data.get("pids", {}).items()
+
+    if len(current_pids) == 0:
+        fg = "yellow"
+        click.secho("record does not have any pids", fg=fg)
+
+    for index, pid in enumerate(current_pids):
+        # BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE, RESET
+        fg = "blue" if index % 2 == 0 else "cyan"
+        click.secho(json.dumps(pid, indent=2), fg=fg)
+
+
+@pids.command("replace")
+@option_pid(required=True)
+@option_pid_identifier(required=True)
+@with_appcontext
+def update_pids(pid: str, pid_identifier: str):
```

Comment:
Why is the name of the command "replace" but the function name is "update"?

Reply:
typo. fixed